stable/kong/values.yaml

```diff
@@ -51,13 +51,33 @@ proxy:
   annotations: {}
   #  service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: "*"
 
+  # Example usage with the parameter "enableTlsOffload"
+  # service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: "3600"
+  # service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0
+  # service.beta.kubernetes.io/aws-load-balancer-ssl-cert: arn:aws:acm:us-east-1:xxxxxxxxxx:certificate/xxxxxxxxx
+  # service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
+  # service.beta.kubernetes.io/aws-load-balancer-backend-protocol: "http"
```

Comment:
Please remove these annotations.

Reply:
Do we want these anywhere as a reference?

Reply:
We can add those in the README as a how to guide for AWS specifically.

Reply:
Removed and put the info into the README.  Looking for feedback to see if what i wrote here makes sense to you and if it is clear.